pkg/metadata/annotations.go

```diff
@@ -46,6 +46,9 @@ type NetIps struct {
 // Service endpoint annotation
 const ServiceEpAnnotation = "opflex.cisco.com/service-endpoint"
 
+// Service scope annotation
+const ServiceScopeAnnotation = "opflex.cisco.com/ext_service_scope"
```

Comment:
I  am wondering if we should call this ServiceContractScopeAnnotation, and also update the annotation string to:
opflex.cisco.com/ext_service_contract_scope

to make things less ambiguous. Also the comment should be a little more descriptive, indicating what is the relevance of this annotation, how it should be used, what are the legal values, and what is the default. I would recommend reflecting this information in the commit message as well.

Reply:
Makes sense, I will add this to code and commit.